### PR TITLE
A few more removals of use of typed, where makes sense. 

### PR DESCRIPTION
**Pull request by helena**
https://github.com/akka/akka/issues/24717
Remaining usages are still needed.

### CODE REVIEW

File: akka-docs/src/main/paradox/typed/persistence.md [hunk 1]
```diff
@@ -242,16 +242,16 @@ Java
...
-As you can see in the above examples this is not supported by typed persistent actors. Instead, the state is
+As you can see in the above examples this is not supported by persistent actors. Instead, the state is
 returned by `eventHandler`. The reason a new behavior can't be returned is that behavior is part of the actor's
 state and must also carefully be reconstructed during recovery. If it would have been supported it would mean
 that the behavior must be restored when replaying events and also encoded in the state anyway when snapshots are used.
-That would be very prone to mistakes and thus not allowed in Typed Persistence.
+That would be very prone to mistakes and thus not allowed in Akka Persistence.
 
-For basic actors you can use the same set of command handlers independent of what state the entity is in,
+For basic actors you can use the same set of command handlers independpedcanent of what state the entity is in,
```
Comment (johanandren) [inline akka-docs/src/main/paradox/typed/persistence.md]: Not quite sure about this word ;)

File: akka-docs/src/main/paradox/typed/style-guide.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Style guide
 
-This is a style guide with recommendations of idioms and pattern for writing Akka Typed actors.
+This is a style guide with recommendations of idioms and pattern for writing Akka actors.
```
Comment (johanandren) [inline akka-docs/src/main/paradox/typed/style-guide.md]: Perhaps a caveat that it it does not cover the classic actor API?

File: akka-docs/src/main/paradox/typed/persistence.md (hunk 1 above)
Comment (helena) [reply akka-docs/src/main/paradox/typed/persistence.md]: Sometimes I make things up to look smart. Fixing ;)

**Review by johanandren — APPROVED** [review]
LGTM

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**